suc/pkg/service/config.go

```diff
@@ -0,0 +1,137 @@
+package service
+
+import (
+	"fmt"
+	"os"
+	"strconv"
+	"strings"
+
+	"github.com/rancher/wins/cmd/server/config"
+	"github.com/sirupsen/logrus"
+)
+
+var (
+	ConfigDirEnvVar      = envVar{varName: "wins_config_dir"}
+	DebugEnvVar          = envVar{varName: "debug"}
+	AgentStringTLSEnvVar = envVar{varName: "strict_verify"}
+)
+
+const (
+	defaultConfigFile = "C:/etc/rancher/wins/config"
+)
+
+type envVar struct {
+	varName string
+}
+
+// get returns the value of the environment variable
+// using os.Getenv. The full name is passed to os.Getenv
+// (i.e. CATTLE_WINS_*)
+func (e *envVar) get() string {
+	return os.Getenv(e.name())
+}
+
+// getSimple returns the value of the environment variable
+// using os.Getenv. The simple name is passed to os.Getenv
+// (i.e. there is no 'CATTLE_WINS_' prefix)
+func (e *envVar) getSimple() string {
+	return os.Getenv(e.simpleName())
+}
```

Comment:
Is this necessary? I feel like we only are ever using the `CATTLE_WINS_` version or raw version in this file, but it doesn't appear like we are mixing them.

Reply:
`STRICT_VERIFY` is platform agnostic and does not use a `CATTLE_*` prefix. I think supporting both formats here makes more sense than changing `STRICT_VERIFY`.

Reply:
IMO I'd remove the simple variants of the functions and just add `cattle_wins_` to the definitions, makes it less confusing of when to use which.

Reply:
Agreed

 I had initially implemented this file with the goal of allowing users to configure other areas of the rancher-wins config file, at which point I think having these functions made more sense. I walked back the idea when I realized how little of it really needs to be changed in order for us to handle the agent tls mode issue, so you're right using raw strings here is better / simpler.